components/automate-gateway/api/iam/v2/request/users.proto

```diff
@@ -9,60 +9,59 @@ message CreateUserReq {
   option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
     json_schema: {
       required: ["id", "name", "password"]
-    }
+    },
+    example: { value: '{"name": "Display Name", "id": "username001rulez", "password": "aSafePassword"}'}
   };
+  // Unique ID. Cannot be changed. Used to sign in.
```

Comment:
aw man this just made me realized we've still got `username` on the sign in page :/
<img width="402" alt="Screen Shot 2020-03-11 at 4 00 25 PM" src="https://user-images.githubusercontent.com/21015366/76471987-731f1700-63b1-11ea-829c-4ff7e6ab1d6d.png">

Reply:
wait why is that bad?

Reply:
Username on the Sign in makes sense, even if it doesn't map directly to the API. Imagine how much frustration we would bring to users if we asked for "Name" instead.